src/main/java/net/sf/rails/game/special/SpecialMultiTileLay.java

```diff
@@ -0,0 +1,236 @@
+package net.sf.rails.game.special;
+
+import net.sf.rails.common.LocalText;
+import net.sf.rails.common.parser.ConfigurationException;
+import net.sf.rails.common.parser.Tag;
+import net.sf.rails.game.*;
+import net.sf.rails.game.state.IntegerState;
+import net.sf.rails.util.Util;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Martin Brumm <dr.martin.brumm@t-online.de>
+ * @date May 20th 2020
+ * This Class is based on the assumption that while you can lay mutiple Tiles, each specific tile can only
+ * be placed once on the map with this SpecialPower for the private with that special power.
+ */
+public class SpecialMultiTileLay extends SpecialTileLay {
+	
+	private String locationCodes = null;
+	private List<MapHex> locations = null;
+	private String name = null;
+	private boolean extra = false;
+	private boolean free = false;
+	private int discount = 0;
+	private String tileIdCodes = null;
+    private String[] tileIds = null;
+    private List<Tile> tiles = null;
+    //Connected means need to be connected to a previous track already on the map and can be reached
+    // via a indefinite train without a blocking token on the route
+    private boolean connected = false;
+    //Constraints can be that the tiles need to face a certain orientation or be connected to each other
+    //Needs to be handled in the specific OperatingRound of that game Variant
+    private String constraints = null;
+    private IntegerState occurred = IntegerState.create(this,"occurred", 0);
+    private int maxOccurrence = 1;
+    
+    /** Tile colours that can be laid with this special property.
+     * Default is same colours as is allowed in a a normal tile lay.
+     * Don't use if specific tiles are specified! */
+    protected String[] tileColours = null;
+
+    /**
+     * Used by Configure (via reflection) only
+     */
+    public SpecialMultiTileLay(RailsItem parent, String id) {
+        super(parent, id);
+    }
+
+
+
+    @Override
+    public void configureFromXML(Tag tag) throws ConfigurationException {
+        super.configureFromXML(tag);
+
+        Tag tileLayTag = tag.getChild("SpecialMultiTileLay");
+        if (tileLayTag == null) {
+            throw new ConfigurationException("<SpecialMultiTileLay> tag missing");
+        }
+
+        locationCodes = tileLayTag.getAttributeAsString("location");
```

Comment:
Might be worth moving the "common" configuration such as `name`, `tileColours`, etc out of here and into the parent class `SpecialTileLay` as that same code is also used by the single tile lay class too, correct? Then we would just leave the unique configuration here.